lib/spack/docs/developer_guide.rst

```diff
@@ -257,22 +251,19 @@ Unit tests
   Implements Spack's test suite.  Add a module and put its name in
   the test suite in ``__init__.py`` to add more unit tests.
 
-:mod:`spack.test.mock_packages`
```

Comment:
Couldn't find this, I assume it's in `var/spack/repos/builtin.mock` now, which we don't generate docs for, so I removed it.